heareval/train_baseline.py

```diff
@@ -0,0 +1,164 @@
+"""
+Train the baseline model on a single multiclass task.
+
+TODO: Some of the boilerplate code is shared with task_embeddings.py
+and can be cleaned up.
+"""
+
+import os.path
+from importlib import import_module
+
+import numpy as np
+import pytorch_lightning as pl
+import soundfile as sf
+import torch
+from torch.utils.data import DataLoader, random_split
+
+import heareval.baseline
+
+from csvdataset import CSVDataset
+from heareval.baseline import RandomProjectionMelEmbedding
```

Comment:
We're using heareval.baseline here, but this should be generic and work with any model that follows our API.